src/backend/XmlHttpRequest.ts

```diff
@@ -396,7 +414,11 @@ export default class XmlHttpRequest extends BaseFileSystem implements FileSystem
   private _requestFileAsync(p: string, type: 'json', cb: BFSCallback<any>): void;
   private _requestFileAsync(p: string, type: string, cb: BFSCallback<any>): void;
   private _requestFileAsync(p: string, type: string, cb: BFSCallback<any>): void {
-    asyncDownloadFile(this.getXhrPath(p), type, cb);
+    if (this._useFetch) {
```

Comment:
It may be verbose to type out the full type of `_asyncDownloadFile`, so you can make an interface instead.

```typescript
interface DownloadFileMethod {
  (p: string, type: 'buffer', cb: BFSCallback<Buffer>): void;
  (p: string, type: 'json', cb: BFSCallback<any>): void;
  (p: string, type: string, cb: BFSCallback<any>): void;
}

// etc

private _asyncDownloadFile: DownloadFileMethod;
```